nncf/experimental/post_training/algorithms/quantization/fast_bias_correction.py

```diff
@@ -0,0 +1,225 @@
+                model_backend, target_point, bias_shift)
+            transformation_layout.register(bias_correction_command)
+
+        quantized_model = self._model_transformer.transform(
+            transformation_layout)
+        return quantized_model
+
+    def get_statistic_points(self, model: ModelType) -> StatisticPointsContainer:
+        model_backend = infer_backend_from_model(model)
+        nncf_graph = NNCFGraphFactory.create(
+            model) if self.nncf_graph is None else self.nncf_graph
+        layers_with_bias_types = PTQLayerMetatypesFactory.get_layers_with_bias_types(
+            model_backend)
+        biased_nodes = nncf_graph.get_nodes_by_metatypes(
+            layers_with_bias_types)
+
+        statistic_container = StatisticPointsContainer()
+
+        for node in biased_nodes:
+            edge_name = node.layer_attributes.input_tensor_names[0]
+            pre_layer_statistic_point = PTQTargetPointFactory.create(
+                model_backend, TargetType.PRE_LAYER_OPERATION, node.node_name, edge_name)
+            post_layer_statistic_point = PTQTargetPointFactory.create(
+                model_backend, TargetType.POST_LAYER_OPERATION, node.node_name)
+            channel_axis = self._channel_axis_by_types[node.node_type]
+
+            self._add_statistic_point(
+                model_backend, statistic_container, pre_layer_statistic_point, channel_axis)
+            self._add_statistic_point(
+                model_backend, statistic_container, post_layer_statistic_point, channel_axis)
+
+        return statistic_container
+
+    def _add_statistic_point(self, model_backend, container, point, axes):
+        stat_collector = PTQMeanStatisticCollectorFactory.create(
+            model_backend,
+            reduction_shape=axes,
+            num_samples=self.number_samples)
+        container.add_statistic_point(StatisticPoint(target_point=point,
+                                                     tensor_collector=stat_collector,
+                                                     algorithm=PostTrainingAlgorithms.FastBiasCorrection))
+
+    def _create_input_blob(self, model_backend, input_shape, input_fp, input_names):
+        input_blob = np.zeros(input_shape)
+        for i, value in enumerate(input_fp):
+            input_blob[:, i] = value
+        input_blob = input_blob.astype(np.float32)
+
+        input_data = PTQNNCFTensorFactory.create(model_backend, input_blob)
+        input_data = {n: input_data for n in input_names}
+        return input_data
+
+    def _calculate_bias_shift(self,
+                              engine,
+                              model,
+                              stat_collector,
+                              input_blob,
+                              channel_axis,
+                              output_fp,
+                              output_names):
+
+        engine.rt_session_options['providers'] = ['OpenVINOExecutionProvider']
```

Comment:
How it is supposed to work across different backends? or this is a part we haven't touched yet.

Reply:
Haven't touched it yet. Maybe it's too early to introduce this. Should we remove it?

Reply:
let's discuss first